index.html

```diff
@@ -198,158 +165,249 @@ <h2>Tonic</h2>
-                class="form-input"
-              />
+              <a href="#"
+                ><img
+                  class="t-social-media"
+                  src="images/twitter.png"
+                  alt="twitter"
+              /></a>
             </li>
             <li>
-              <label for="mail"></label>
-              <input
-                type="email"
-                id="mail"
-                name="user_mail"
-                placeholder="your email "
-                class="form-input"
-                required
-              />
+              <a href="#"
+                ><img
+                  src="images/linkedin.png"
+                  alt="linkedIn"
+              /></a>
             </li>
             <li>
-              <label for="msg"></label>
-              <textarea
-                id="msg"
-                name="user_message"
-                maxlength="500"
-                placeholder="write something here"
-                class="form-input"
-                rows="5"
-                required
-              ></textarea>
+              <a href="#"
+                ><img
+                  src="images/medium.png"
+                  alt="medium"
+              /></a>
             </li>
             <li>
-              <button class="btn contact-btn" type="submit">Get In Touch</button>
+              <a href="#"
+                ><img
+                  src="images/github.png"
+                  alt="github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img class="v-social-media" src="images/vec.png" alt="vector"
+              /></a>
             </li>
           </ul>
-        </form>
+          <button type="button" class="click-btn">Get my resume</button>
+        </section>
+        <section>
+          <ul class="lang">
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Languages</p>
+                <img src="images/arrows-down.png" alt="arrow_bottom" />
+              </div>
+              <ul class="languages">
+                <li>
+                  <img src="images/js.png" alt="JavaScript" />
+                  <a href="#">JavaScript</a>
+                </li>
+                <li>
+                  <img src="images/html..png" alt="CSS" />
+                  <a href="#">HTML</a>
+                </li>
+                <li>
+                  <img src="images/css..png" alt="CSS" />
+                  <a href="#">CSS</a>
+                </li>
+              </ul>
+            </li>
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Frameworks</p>
+                <img src="images/arrow.png" alt="arrow_bottom" />
+              </div>
+              <hr />
+            </li>
+            <li class="lang-exp">
+              <div class="no-1">
+                <p>Skills</p>
+                <img src="images/arrow.png" alt="arrow_bottom" />
+              </div>
```

Comment:
- Please properly link your arrow button image so it shows up on the webpage. 

##### Your design 
![Screenshot from 2022-07-16 10-06-00](https://user-images.githubusercontent.com/58771507/179348445-2df7a3f6-07cd-42a4-8882-a78cb581def8.png)